processing_r/processing/utils.py

```diff
@@ -166,6 +166,34 @@ def create_descriptive_name(name):
         """
         return name.replace('_', ' ')
 
+    @staticmethod
+    def is_valid_r_variable(variable: str) -> bool:
+        """
+        Check if given string is valid R variable name.
+        :param variable: string
+        :return: bool
+        """
+
+        # only letters a-z, A-Z, numbers, dot and underscore
+        x = re.search("[a-zA-Z0-9\\._]+", variable)
+
+        result = True
+
+        if variable == x.group():
+            # cannot start with number or underscore
+            x = re.search("^[0-9|_]", variable)
+            if x:
+                result = False
```

Comment:
Minor thing. But we should return here and avoid the extra expense of an unnecessary regex test on the next line

Reply:
These two checks can be simplified into a single regex. Hopefully, I managed to do it correctly.  Waiting for travis to check out.